tests/routers/v1/application_test.py

```diff
@@ -88,11 +87,7 @@ def test_data_selected_version_returns_400(client: TestClient, seed_application)
     assert response.status_code == 400
 
 
-@patch("laa_crime_application_store_app.internal.notifier.Notifier.notify")
-def test_post_application_returns_200(
-    mock_notify, client: TestClient, dbsession: Session
-):
-    mock_notify.return_value = True
+def test_post_application_returns_200(client: TestClient, dbsession: Session):
```

Comment:
this is tricky because to my mind this test should fail as you have no roles (roles is an empty list) - The test should pass with a provider role and fail with caseworker or anything else as here we are mocking what happens when there is an authenticated user.
The issue is that we have methods that allow us to continue if there are no roles.

Reply:
We may have to feature flag this.